data/json/items/battery.json

```
@@ -336,5 +336,62 @@
     "looks_like": "battery",
     "flags": [ "NO_SALVAGE", "NO_UNLOAD" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 2500 } } ]
+  },
+  {
+    "id": "small_lipo_battery",
+    "type": "MAGAZINE",
+    "category": "spare_parts",
+    "name": { "str": "small LiPo battery pack", "str_pl": "small LiPo battery packs" },
```

@LucasHoage I think they just mean something like this, a comment telling translators what LiPo is
```suggestion
    "name": { "str": "small LiPo battery pack", "str_pl": "small LiPo battery packs", "ctxt": "LiPo is lithium polymer" },
```

---

"str_pl" is not necessary here since the plural form can be automatically generated.
```suggestion
    "name": { "str": "small LiPo battery pack", "ctxt": "LiPo is lithium polymer" },
```

---

@anothersimulacrum: Yes, exactly, I was in no way suggesting that the player presented name be changed!

---

10-4 buddy sorry about all that confusion I'm still new and don't know all the ins and outs of what's referenced.

Sometimes I'm oblivious like I said too. Maybe even a little obnoxious.

I still want you to follow up on the other parts of the discussion about removable batteries. I think that does need a resolution. Either today or down the road. I don't necessarily see what I'm supposed to see so feedback is very important to me.

---

Changes made.